### PR TITLE
Minutes last meeting

### PR DESCRIPTION
**Pull request by teresanunez**

### CODE REVIEW

File: 20181213/MINUTES.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+     
+2. Overview of current developments
+	* Check motor limits before starting scan; it is probably finished,
+	last requested changes are done. There are some problems with travis,
+	some scans give timeouts, they have to be solved. But it can be
+	probably integrated after solving this and checking the last changes.
+	* Asynchronous Logstash Communication - PR#895. The last modifications
+	were done, a last test would be done and then it would be ready
+	for being integrated.
+        At MAXIV the log files are written  using filebeat from logstash, the
```
Comment (reszelaz) [inline 20181213/MINUTES.md]: ...log files are read with filebeat and send to logstash...

File: 20181213/MINUTES.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+     this happens again they modified spock, so that it sends a message if
+     the standard macros are not overloaded.
+     
+   * DESY:
+     - in one beamline the measurement group stays for ever in MOVING at
+     some point of the scan. One has to stop the measurement group, calling
+     its Stop command from some other client, and the scan continues without
+     problems. A script was done for detecting this and solving it stoping
+     the measurement group. It happens several times per day. It will be
+     investigated in detail next week.
```
Comment (reszelaz) [inline 20181213/MINUTES.md:73]: Here we need to check if the measurement groups stays in MOVING and all its channels statys in MOVING as well. For example using Jive.

File: 20181213/MINUTES.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+	for being integrated.
+        At MAXIV the log files are written  using filebeat from logstash, the
+	configuration is easy and does not need modifications in Sardana.
+	Antonio offers this as another alternative.
+        The documentation will be put in the wiki because it is not a
+	Sardana feature.
+	* SEP18, the development is done. The PR is fast reviewed,
+	there are still some decisions to take. About the controller API,
+	if there is no comments the option 1 will be used. The rest of the
+	things are implemented. Stop channels configured with hardware
```
Comment (reszelaz) [inline 20181213/MINUTES.md]: software trigger/gate

**Review by reszelaz — APPROVED** [review]
Very complete! Just three small suggestions.

**Review by daneos — APPROVED** [review]
Looks great, thanks!

**Comment by teresanunez** [comment]
Hi, should i merge this?. @amilan, do you agree with the minutes?.

**Review by amilan — APPROVED** [review]

**Comment by amilan** [comment]
Hi @teresanunez, minutes are OK. Thanks a lot, and sorry for the late answer (blame Christmas holidays :P)

**Comment by teresanunez** [comment]
@amilan, no problem, thanks :)

**Pull request merged**